art_show/templates/emails/art_show/payment_confirmation.txt

```diff
@@ -1,7 +1,6 @@
 {{ app.attendee.first_name }},
 
 Your payment of ${{ app.total_cost }} for space in the {{ c.EVENT_NAME }} Art Show has been received. You will have
-{{ app.panels }} panels and {{ app.tables }} tables available to display your art. The Art Show director will continue
-to be in touch with you to keep you informed of Art Show rules and procedures.
+{{ app.panels }} general panel{{ app.panels|pluralize }}, {{ app.panels_ad}} adult panel{{ app.panels_ad|pluralize }}, {{ app.tables }} general table{{ app.tables|pluralize }}, and {{ app.tables_ad}} adult table{{ app.tables_ad|pluralize }} available to display your art. {% if app.delivery_method == c.BY_MAIL %} Your mail-in fee was ${{ c.ART_MAILING_FEE }}. {% endif %}The Art Show director will continue to be in touch with you to keep you informed of Art Show rules and procedures.
```

Comment:
This will still show a line break between "have" and the number of panels. Is this okay?

Reply:
not ideal but easy to fix